### DISCUSSION TITLE
Does web sockets work outside the general network.

### DISCUSSION THREAD

**Discussion by mccla5214** [Palakis/obs-websocket#828] [General]
I'm working with one of the suggested client applications written in C##. Right now it won't create a connection except for using "ws://127.0.0.1:4444". Before I start down the road of finding out why it won't create the connection, I wanted to see if the web sockets plug-in works from outside the local network.

Thanks for a great plug-in. This is going to open up a whole world of possibilities for me when I get it working.

Mike

**Comment by chrislowth** [comment]
The port number (4444) can be configured using Tools -> WebSockets Server Settings.

The IP address cant be changed but there is a guide on using tunnelling to overcome this restriction. You can read it at ...

https://github.com/obs-websocket/obs-websocket/blob/4.x-current/SSL-TUNNELLING.md
